src/main/java/org/levelup/labradoodle/controllers/HomeController.java

```diff
@@ -69,4 +69,22 @@ public DishesDto getDishById(@PathVariable(value = "id") int id) {
         }
         return typeDishes;
     }
+
+    /**
+     * This method return Dish (WEB model) from DB by id. Current version work with mock
+     * @author Barkovskiy Alexandr
+     * @return Dish
+     */
+    @ResponseBody
+    @RequestMapping(value = "/getFirst20",method = RequestMethod.GET)
+    public List<DishesDto> getFirst20Dishes(){
+        List<DishesDto> dishesDtos = null;
+        try {
+            dishesDtos = appService.getFirst20Dishes();
+        }
+        catch (Exception e){
+            System.out.println(e);//Maybe we need write it in log
```

Comment:
we certainly need write it into logger.. And plz move exceptions catching at service layer